x-pack/plugins/monitoring/public/alerts/lib/replace_tokens.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { Fragment } from 'react';
+import moment from 'moment';
+import { EuiLink } from '@elastic/eui';
+import {
+  AlertMessage,
+  AlertMessageTimeToken,
+  AlertMessageLinkToken,
+  AlertMessageDocLinkToken,
+} from '../../../server/alerts/types';
+// @ts-ignore
+import { formatTimestampToDuration } from '../../../common';
+import { CALCULATE_DURATION_UNTIL } from '../../../common/constants';
+import { AlertMessageTokenType } from '../../../common/enums';
+import { Legacy } from '../../legacy_shims';
+
+// interface Props {
+//   alertMessage: AlertMessage;
+// }
+// export const Tokenize: React.FC<Props> = (props: Props) => {
+
+// }
+
+export function replaceTokens(alertMessage: AlertMessage): JSX.Element | string | null {
+  if (!alertMessage) {
+    return null;
+  }
+
+  let text = alertMessage.text;
+  if (!alertMessage.tokens) {
```

Comment:
nit: Just in case it's an empty array 
```suggestion
  if (!alertMessage.tokens || !alertMessage.tokens.length) {
```

Reply:
Sure!